src/diffusers/image_processor.py

```diff
@@ -523,6 +548,105 @@ def postprocess(
             return image[:, :, :, :3], image_depth
 
         if output_type == "pil":
-            return self.numpy_to_pil(image), self.numpy_to_depth(image)
+            return self.numpy_to_pil(image), self.numpy_to_pil_depth(image)
         else:
             raise Exception(f"This type {output_type} is not supported")
+
+    def preprocess(
```

Comment:
Hmm is this really needed?

Reply:
the preprocess? or the raise?

Reply:
Ah sorry I misread here, actually this look ok to me - @yiyixuxu wdyt?

Reply:
yeah fine with me too!